journald/Makefile

```
@@ -0,0 +1,23 @@
+CC=gcc
+CFLAGS=-O0 -g -pipe
+CFLAGS+=$(shell python3.13-config --cflags)
```

```suggestion
CFLAGS+=$(shell python3-config --cflags)
```

Should we just use `python3` here? The readme says "copy of Python 3.8.0rc1 or later" which I'm guessing is copy-pasted from one of the others, but there's no reason to pin to 3.13 in the Makefiles, right?

---

done